src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -236,6 +242,7 @@ public boolean performInternal(Run<?, ?> run, FilePath workspace, Launcher launc
       env.put("SONAR_RUNNER_HOME", sri.getHome());
     }
     addTaskArgument(args);
+    addAdditionalOptions(args);
     args.add("-e");
```

Comment:
How does sonar-runner react if we pass "-e" several times? For example if user put -e in extra args?